uv.lock

```diff
@@ -1660,30 +1715,66 @@ wheels = [
 
 [[package]]
 name = "sphinx"
-version = "7.4.7"
+version = "8.1.3"
```

Comment:
Is there a reason we have this at version 7.4.7 in every project except for craft-store?